backend/t4cclient/routes/repositories/users.py

```diff
@@ -9,6 +10,7 @@
     is_admin,
     verify_repository_role,
     verify_write_permission,
+    check_username_not_in_repository,
```

Comment:
The import is at the wrong location. Could you please apply `isort` for the file?